reference/tugboat-configuration/index.md

```diff
@@ -0,0 +1,194 @@
+
+Specifies the path where the git repository should be cloned if `checkout: true`
+is set. If this path already exists, the clone will fail.
+
+## `commands`
+
+* **Type:** List
+* **Default:** _no default_
+* **Required:** No
+
+A set of commands to run during various points in a preview's lifecycle. These
+commands are divided into the following stages. Each stage consists of a list of
+commands. Each command is run in its own context, so things like changing
+directories does not "stick" between commands. If that behavior is requried, an
+external script should be called.
+
+| Stage  | Description                                                                                            |
+| :----- | :----------------------------------------------------------------------------------------------------- |
+| init   | Commands that set up the basic preview infrastructure, such as required packages or tools              |
+| update | Commands that import data or other assets into a service, such as a database or image files            |
+| build  | Commands that build or generate the site, such as compiling Sass or running database updates from code |
+| ready  | Commands that indicate that a service is "ready", such as checking for a listening TCP port            |
+
+The `init`, `update`, and `build` stages are related as follows
+
+* When a preview is created, the commands in `init` are run, followed by the
+  commands in `update`, and finally the commands in `build`.
+
+* When a preview is refreshed, the commands in `update` are run, followed by the
+  commands in `build`.
+
+* When a preview is created from a base preview, only the commands in `build`
+  are run.
+
+## `default`
+
+* **Type:** Boolean
+* **Default:** `false` (`true` if only one service is defined)
+* **Required:** Yes, if more than one service is defined.
+
+Whether this is the default service for a preview. The default service is where
+incoming HTTP requests to the preview URL are routed. Setting this to true also
+implies `expose: 80` and `checkout: true` unless those attributes are explicitly
+set otherwise.
+
+## `depends`
+
+* **Type:** List
+* **Default:** _no default_
+* **Required:** No
+
+Defines the order in which commands are executed between the defined services.
+If one service has a dependency on another, it will wait for that service's
+commands to run before running its own commands. If not set, there is no
+guaranteed order in which services will execute their commands relative to other
+services.
+
+## `domain`
+
+* **Type:** String
+* **Default:** _no default_
+* **Required:** No
+
+A custom domain to use when generating URLs for the service. If `example.com` is
+used, for example, URLs for the service will resemble
+`http://pr123-token.example.com` or `http://preview.example.com/pr123-token/`,
+depending on the other configuration values. Note that using a custom domain
+will result in browsers issuing SSL/TLS certificate warnings when combined with
```

Comment:
Can the customer make a change in their own setup to prevent certificate warnings, or is this unavoidable when using a custom domain and HTTPS enabled?

Reply:
This is just the way it is for a custom domain with HTTPS